### ISSUE TITLE
[Question] Future of this App?

### ISSUE THREAD

**Issue by alphanerdgen** [jameskokoska/Cashew#508]
I'm new to Cashew, and liking it so far. I was using Ivy Wallet previously until its dev decided to ditch it for spreadsheets. What a shame!

Now, the cost for upgrading to Cashew Pro seems reasonable, but the only thing stopping me from shelling out is the question mark over the longevity of foss apps. What if I pay today but tomorrow you also decide to ditch your app for spreadsheets (or whatever)?

So, what's your vision/roadmap for Cashew? How long do you intend to develop it?

**Comment by jameskokoska** [comment]
Hi there! I actually use Cashew myself daily, reason being ~ I think it does a better job than spreadsheets (for me at least!). I made Cashew with the main goal to be more efficient and user friendly than that of a spreadsheet. 

I also have a major update coming soon (trying to get it out before the holidays) however it’s currently in personal beta testing. Releases have slowed down because more users = more backlash for smaller bugs (so testing new features takes longer to make sure they are bug free). 
Hope this makes sense :)

**Comment by jameskokoska** [comment]
Yes, I am hoping to get a public beta out sometime within a week. Please be patient as development takes time to ensure a bug free release. I still use the app every day and I am still actively developing this app almost daily in my free time.

**Comment by jameskokoska** [comment]
See https://github.com/jameskokoska/Cashew/issues/725 for Beta Testing instructions and information
